addons/website_event_sale/controllers/main.py

```diff
@@ -91,3 +92,52 @@ def _prepare_event_values(self, name, event_start, event_end, address_values=Non
                 }]]
             })
         return values
+
+    @route(['/website_event_sale/ticket-price-info/<int:event_ticket_id>'],
+           type='json', auth="public", website=True, sitemap=False)
+    def get_ticket_price_info(self, event_ticket_id, quantity=1):
+        """
+        :param: int event_ticket_id: id of an event ticket
+        :param: int quantity: optional quantity (default 1)
+        :return: a dict containing detailed unit price of the ticket for the given quantity
+        """
+        pricelist = request.website.pricelist_id
+        event_ticket = request.env['event.event.ticket'].browse(event_ticket_id)
+        company_id = event_ticket.event_id.sudo().company_id
+        ticket_currency = company_id.currency_id
+        ctx = dict(quantity=quantity)
+        if pricelist:
+            ctx.update(pricelist=pricelist.id)
+            currency_id = pricelist.currency_id
+            discount_policy = pricelist.discount_policy
+        else:
+            currency_id = ticket_currency
+            discount_policy = None
+
+        event_ticket = event_ticket.with_context(ctx)
+
+        if bool(request.env.user.has_group('account.group_show_line_subtotals_tax_included')) \
+                or not bool(request.env.user.has_group('account.group_show_line_subtotals_tax_excluded')):
```

Comment:
:thinking:

Reply:
“group_show_line_subtotals_tax_excluded and group_show_line_subtotals_tax_included are opposite, so we can assume exactly one of them will be set, and not the other.”
It could be simplified to only one condition. As the worst error is to display excluded tax when we must display included tax. I have simplified it to not having the “group_show_line_subtotals_tax_excluded” group ; basically by default we display it tax included if not specified otherwise by the presence of  group_show_line_subtotals_tax_excluded.